molecule/default/molecule-no-systemd.yml

```diff
@@ -16,8 +16,24 @@ lint: |
   ansible-lint
   flake8
 platforms:
-  - image: fedora:35
-    name: fedora35
+  - image: amazonlinux:2
+    name: amazonlinux2
+    platform: amd64
+  - dockerfile: Dockerfile_debian_9.j2
+    image: debian:stretch-slim
+    name: debian9
+    platform: amd64
+  - image: debian:buster-slim
+    name: debian10
+    platform: amd64
+  - image: debian:bullseye-slim
+    name: debian11
+    platform: amd64
+  - image: debian:bookworm-slim
+    name: debian12
+    platform: amd64
+  - image: kalilinux/kali-rolling
+    name: kali
```

Comment:
This jazz should not have been added.  This Ansible role only supports Fedora, as denoted in the role metadata.
```suggestion
```